tensorboard/plugins/image/images_demo.py

```diff
@@ -53,6 +53,7 @@
 (IMAGE_WIDTH, IMAGE_HEIGHT) = (640, 480)
 _IMAGE_DATA = None
 
+tf.compat.v1.disable_eager_execution()
```

Comment:
`tf.compat.v1.disable_v2_behavior` might be more correct to call.

Reply:
done